tasks/config.yml

```diff
@@ -89,6 +89,56 @@
      dest: /etc/nginx/sites-available/bigbluebutton
   notify: reload nginx
 
+- name: disable CPUSchedulingPolicy in FreeSWITCH service
+  lineinfile:
+    path: /lib/systemd/system/freeswitch.service
+    regexp: '^CPUSchedulingPolicy=rr'
+    line: '#CPUSchedulingPolicy=rr'
+  notify: restart freeswitch
+  when: not bbb_cpuschedule
```

Comment:
Please also make this option revertable...

Reply:
This has been now fixed in: https://github.com/n0emis/ansible-role-bigbluebutton/pull/3/commits/669fe9562e847ccfd9f05173a74465a458fb46a2